### DISCUSSION TITLE
Carla on Unreal 5.0 early access

### DISCUSSION THREAD

**Discussion by SanggunLee** [carla-simulator/carla#4394] [Show and tell]
Im working for
Carla dev branch on unreal5.0

**Comment by SanggunLee** [comment]
1. Fix Carla.Build.cs file 
	remove win32
2. Fix Unreal/CarlaUE4/Makefile
	change it
	PROJECTBUILD = ~~"(UNREALROOTPATH)/En......"~~ "$(UNREALROOTPATH/Engine/Binaries/DotNET/~~UnrealBuildTool.exe~~/UnrealBuildTool/UnrealBuildTool
3. PhysX changed

**Comment by DumitruAlin** [comment]
Hi,

Are there any updates or info (estimations) for when Carla will be ready to use in Unreal Engine 5?

**Reply by SanggunLee** [reply]
Unfortunately, I no longer work on it.

**Reply by deo-abhijit** [reply]
@DumitruAlin  you might be interested in this : https://github.com/carla-simulator/carla/issues/5318#issuecomment-2051534295
